plugin-common/src/test/java/org/opennms/plugins/cloud/config/ConfigurationManagerTest.java

```diff
@@ -200,4 +204,31 @@ public void shouldCallConfigureIfAuthenticatedOrConfigured() {
         assertEquals(CONFIGURED, cm.getStatus());
         verify(grpc, times(1)).initGrpc(any()); // this is done in configure()
     }
+
+    @Test
+    public void shouldAbleToDeactive() {
+        GrpcExecutionHandler grpcHandler = new GrpcExecutionHandler(new CloudLogService(cloudLogServiceConfig));
+        TsaasStorage grpc = spy(new TsaasStorage(new TsaasConfig(1, 1), grpcHandler));
+        List<GrpcService> serviceList = Collections.singletonList(grpc);
+        ConfigurationManager cm = new ConfigurationManager(config, clientConfig, clientConfig, mock(RegistrationManager.class),
+                info,
+                serviceList);
+        cm.initConfiguration("something");
+        cm.configure();
+        // should be normal
+        cm.checkConnection();
+
+        cm.deactivateKeyConfiguration();
+
+        assertEquals(DEACTIVATED, cm.getStatus());
+        Exception statusException = null;
+        try {
```

Comment:
can be replaced with assertThrows(()-> cm.checkConnection(), StatusRuntimeException.class)